fdbclient/include/fdbclient/BlobGranuleCommon.h

```diff
@@ -358,10 +358,11 @@ struct BlobManifestTailer {
 	int64_t totalRows;
 	int64_t totalSegments;
 	int64_t totalBytes;
+	Optional<BlobGranuleCipherKeysMeta> cipherKeysMeta;
```

Comment:
nit: can you please document the encryption guarantees; such as: 

`All manifest files are currently encrypted using `default_domain` and with a single encryption key.  
TODO: Extend `domain_aware` encryption semantics to manifest file(s)`

Reply:
fixed. thanks